assets/css/extended/zzz-footnote-preview.css

```
@@ -0,0 +1,179 @@
+}
+
+/* The note's text. Tame block margins so a one-line note isn't padded
+   like a full article paragraph, and keep long links from overflowing. */
+.fn-popover__body > :first-child { margin-top: 0; }
+.fn-popover__body > :last-child { margin-bottom: 0; }
+.fn-popover__body p { margin: 0 0 0.4rem; }
+.fn-popover__body a {
+  color: var(--color-accent, #2e352e);
+  text-decoration: underline;
+  text-underline-offset: 2px;
+  word-break: break-word;
```

Stylelint (17.13.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)